ACPs/77-reinventing-subnets/README.md

```diff
@@ -28,38 +28,38 @@ Although the fee paid to the Primary Network to operate a Subnet does not go up
 
 Elastic Subnets, introduced in [Banff](https://medium.com/avalancheavax/banff-elastic-subnets-44042f41e34c), enabled Subnet creators to activate Proof-of-Stake validation and uptime-based rewards using their own token. However, this token is required to be an ANT (created on the X-Chain) and locked on the P-Chain. All staking rewards were distributed on the P-Chain with the reward curve being defined in the `TransformSubnetTx` and, once set, was unable to be modified.
 
-With no Elastic Subnets live on Mainnet, it is clear that Permissionless Subnets as they stand today could be more desirable. There are many successful Permissioned Subnets in production but many Subnet creators have raised the above as points of concern. In summary, the Avalanche community could benefit from a more flexible and affordable mechanism to launch Permissionless Subnets.
+With no Elastic Subnets live on Mainnet, it is clear that permissionless, sovereign Subnets as they stand today could be more desirable. There are many successful permissioned Subnets in production but many Subnet creators have raised the above as points of concern. In summary, the Avalanche community could benefit from a more flexible and affordable mechanism to launch sovereign, layer 1 blockchain networks.
```

Comment:
Thoughts on adding a sentence to the motivation explaining that after ACP-77, "Subnet" validator sets are no longer a subset of the Primary Network validator set, and that starting with ACP-77 and going forward, these sovereign networks on Avalanche will be referred to as Layer 1s or L1s to reflect this?

Reply:
We could then also get rid of the "open questions" section 👍

Reply:
I pretty much replaced the open questions with the explicit renaming reasoning. I could move it to this section for more clarity as well